lib/ask-stack-result-view.coffee

```diff
@@ -198,7 +215,17 @@ class AskStackResultView extends ScrollView
           atom.workspace.activatePreviousPane()
           # editor = atom.workspace.activePaneItem
           editor = atom.workspace.getActivePaneItem()
-          editor.insertText(code)
+          if qa == true
+            editor.insertText("Insert from Stack Overflow", {select: true})
+            editor.toggleLineCommentsInSelection();
+            editor.insertNewlineBelow();
+            editor.insertText("Author: #{author_name} - #{author_src}", {select: true})
+            editor.toggleLineCommentsInSelection();
```

Comment:
Did you try to find a way to only do this once for the whole selection? Every action here is saved in history so if you want to remove the inserted code you have to undo (cmd+z) multiple times.